Dockerfile

```diff
@@ -33,7 +33,8 @@
 # Stage 1 - Acquire the CARMA source as well as any extra packages
 # /////////////////////////////////////////////////////////////////////////////
 
-FROM usdotfhwastol/autoware.ai:carma-system-3.5.0 AS base-image
+FROM usdotfhwastol/autoware.ai:carma-system-3.6.0 AS base-image
+FROM usdotfhwastol/autoware.ai:zephyr AS base-image
```

Comment:
You removed the <<<< but didn't actually resolve the conflict since both these lines are still present. Please clean it up so only the 3.6.0 line exists